Content.Server/Shuttles/Components/ShuttleComponent.cs

```diff
@@ -66,9 +66,9 @@ public sealed partial class ShuttleComponent : Component
         /// Damping applied to the shuttle's physics component when not in FTL.
         /// </summary>
         [DataField("linearDamping"), ViewVariables(VVAccess.ReadWrite)]
-        public float LinearDamping = 0.05f;
+        public float LinearDamping = 0.00f;
```

Comment:
Im not sure if its a good idea to have this 0 as possible lag issue?
Also in general this should be a per ship toggle options

Reply:
yeah in testing things infinitely floating off does have some issues, that said the current value of .05 feels like way too much, .002 has a lot of drift while definitely slowing vehicles down over a while so I'll probably switch it over to that for the PR